src/main/java/com/endava/soa_5/exception_and_io/FileConverter.java

```diff
@@ -0,0 +1,90 @@
+package com.endava.soa_5.exception_and_io;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.compile;
+
+public class FileConverter {
+
+    private String outputFileName;
+    private final String FILE_NAME;
+    private final String RESOURCE_PATH;
+    private final List<String> SPLIT_STRING_LIST;
+    private List<String> contents;
+
+    FileConverter(String FILE_NAME) {
+        this.FILE_NAME = FILE_NAME;
+        RESOURCE_PATH = "src/main/resources/";
+        SPLIT_STRING_LIST = new ArrayList<>();
+    }
+
+    public List<String> convertFileContentIntoList() {
+        try {
+            contents = Files.readAllLines(Paths.get(RESOURCE_PATH + FILE_NAME).toAbsolutePath());
+
+        } catch (IOException ioException) {
+            System.out.println(ioException.getMessage());
+        }
+        return contents;
+    }
+
+    public List<String> splitList() {
+        String[] temporalArray;
+        for (String content : convertFileContentIntoList()) {
+            temporalArray = content.split("[ ]|[.]|[,]|[ \\t]");
```

Comment:
Because the software is changing in time, after few weeks you may get a request to define another format for your file information. 
So, instead of going in the project tree and search for the pattern you're splitting the rows - maybe you can define this regexp pattern in a class - Constants, for example ? It will be easy to find and edit the code.

Reply:
Done!